codegen/smithy-aws-go-codegen/src/main/java/software/amazon/smithy/aws/go/codegen/RestJsonProtocolGenerator.java

```diff
@@ -44,55 +61,305 @@
     }
 
     @Override
-    protected void generateDocumentBodyShapeSerializers(GenerationContext context, Set<Shape> shapes) {
+    protected void generateOperationDocumentSerializer(
+            GenerationContext context, OperationShape operation
```

Comment:
Nit: These params should be on separate lines.

Reply:
Intellij is smart to wrap them, but not wrap them in the right format :(